Specs/DateTime/English/DateTimeModel.json

```diff
@@ -5900,5 +5900,106 @@
         }
       }
     ]
+  },
+  {
+    "Input": "Cortana, can you please set something up for the week of the 18th.",

```

Comment:
Can you add some "week of the" case to the DateRange specs?

Reply:
Added.